### PR TITLE
Small map / bug fixes

### LINKED ISSUE #1612: Teleporter Access Button gives Access Denied with CSO access; Teleport Reserve Access Button is set to Engine Access.
The teleporter access button to open the teleporter shutter (which is only accessible through the CSO's office at round start) gives an Access Denied message when used with CSO's default access.

### LINKED ISSUE #1707: Attempting to Speak Neohongo fails
As the title says, when attempted to use ,j to speak in neohongo, it simply reads out as such.
image
The code shows that this is correct, neohongo is spoken with ,j
image
The only way to actually speak the language is to set it as default and continue speaking.

### LINKED ISSUE #1724: Cleaning Items
I love the idea of the floor becoming dirty over time, but when theres no custodians it will get very dirty, very fast. Thinking maybe to remedy this, add cleaning stations that the general public can get to?

### PR DESCRIPTION
**Pull request by Valanthe500**
## About The Pull Request
As there's still no sign of Nestor giving us what's needed for the upstream merge despite my constant badgering, figured I might as well get a bit of work done:

- Removes floating holo-posters in cryo area
image
image

- moves intercoms away from cryotubes / beds (this prevents a bug where spawning in will equip nearby intercoms into your suit storage)

- adds cleaning lockers that have simple cleaning supplies, to ease the mess with no custodians (fixes #1724)
image

- adds cable knots under the shield generators in the Engineering entrance, allowing them to be used without having to tear up floorboards. (Also unmodified the vars in those gens to make it so Engineering could use them, they required Laz access for some reason?)
image

- Fixes #1707  Neohongo now mapped to "n"

- Fixes #1612 Set Teleporter access button req_access to 17- Teleporter (now same access as airlock into room)

- Removed a whole bunch of lights in maintenance to darken it, save power (seriously, one of the maint sections in engineering was using 3,000W just on lighting), and better delineate the "habited" areas of the ship from the "uninhabited."
image
image

- Re-arranged the disposal pipe section near the Biogenerator (one pipe was overlapping another, something that doesn't happen anywhere else on the ship)
image
image

- Adds a disposal chute to the First Section Disposals area (near Perma) - allows trash to be sent to normal disposals if desired.
image
The disposal unit
image
A little EVA jump to help discourage careless use.
## Why It's Good For The Game
These are just a series of small bugfixes and some map adjustments based on community feedback. Plus an addition of my own (the disposal chute) 

## Changelog
:cl:
add: Added new janitorial closets
add: added a new disposal chute from security disposals into the main disposal line.
del: Removed a bunch of lights from maintenance, 
fix: fixes Neohongo - now mapped to "n"
/:cl:

### CODE REVIEW

File: zzz_modular_eclipse/code/game/objects/structures/crates_lockers/closets/utility_closets.dm [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//A basic closet with some simple cleaning supplies to help keep the ship clean if there are no custodians
+/obj/structure/closet/mopcloset
+	name = "\improper Janitorial closet"
```
Comment (EvilJackCarver) [inline zzz_modular_eclipse/code/game/objects/structures/crates_lockers/closets/utility_closets.dm:3]: Why is this capitalised? That'll make it say "This is a Janitorial closet"...

Comment (Valanthe500) [reply zzz_modular_eclipse/code/game/objects/structures/crates_lockers/closets/utility_closets.dm:3]: I'm gonna blame habit on that

Comment (EvilJackCarver) [reply zzz_modular_eclipse/code/game/objects/structures/crates_lockers/closets/utility_closets.dm:3]: DANGIT BOBBEH

HOW MANY TIMES 'V I GOT TA TELL YOU ABOUT PROPANE AND PROPANE ACCESSORIES

File: maps/CEVEris/_CEV_Erida.dmm
Comment (EvilJackCarver) [inline maps/CEVEris/_CEV_Erida.dmm:1]: You have a merge conflict somewhere in this file. Check your git tools; I can't resolve it from GitHub.

**Review by EvilJackCarver — CHANGES_REQUESTED** [review]

File: maps/CEVEris/_CEV_Erida.dmm
Comment (Valanthe500) [reply maps/CEVEris/_CEV_Erida.dmm:1]: I have no idea what or how I broke the map so completely.

It's fixed now, I hope, I tested it locally and nothing is exploding anymore.

**Comment by EvilJackCarver** [comment]
Seeing some possible corruption here. I'm still looking.

image

**Comment by EvilJackCarver** [comment]
Finished checking, that's the only odd thing that stood out to me.

**Comment by Valanthe500** [comment]
Yeah missed that one, forgot to look under the roach spawner, fixed now.

I did another once over of that area again, think I got everything now.

**Review by EvilJackCarver — APPROVED** [review]

**Pull request merged**